### PR TITLE
Papa Glider Updates

### PR DESCRIPTION
**Pull request by cdobs**
-gp_276 recovered on TY-1902. This cruise was added to the CruiseInfo.csv in the most recent push to the Pioneer branch
-gp_469 recovered on SKQ20190514. This cruise was added to the CruiseInfo.csv in the most recent push to the Pioneer branch

### CODE REVIEW

File: cruise/CruiseInformation.csv [hunk 1]
```diff
@@ -1,158 +1,162 @@
...
+WJ20190228,M/V Warren Jr.,2019-02-28T17:00:00,2019-03-02T13:00:00,"Pioneer 11c, AUV deployments, glider cp_339 recovery"
+SKQ201905S,R/V Sikuliaq,2019-03-24T00:00:00,2019-03-24T00:00:00,Endurance opportunistic glider ce_383 recovery
+EK20190403,R/V Elakha,2019-04-03T15:00:00,2019-04-03T22:00:00,ce01issp_00012 and ce02shsp_00012 deployments
+AR-34,R/V Neil Armstrong,2019-04-04T15:10:00,2019-04-24T11:36:00,Pioneer 12
+ML20190417,R/V Miss Linda,2019-04-07T15:07:00,2019-04-17T22:00:00,Endurance glider ce_327 recovery
+SKQ201910S,R/V Sikuliaq,2019-04-19T20:00:00,2019-04-28T15:46:47,Endurance 11A
+OC1905C,R/V Oceanus,2019-05-03T20:00:00,2019-05-08T00:00:00,Endurance 11B
+BK20190504,M/V Blackhawk,2019-05-04T09:00:00,2019-05-04T20:00:00,Pioneer cp_340 recovery
+TS20190516,F/V Tenacious II,2019-05-16T12:00:00,2019-05-17T12:00:00,Pioneer 12a glider deployment
+SKQ20190514,R/V Sikuliaq,2019-05-14T00:00:00,2019-05-27T00:00:00,Papa Glider gp_469 recovery on the San Diego - Seward transit
```
Comment (rtravis-whoi) [inline cruise/CruiseInformation.csv]: Reviewed by RT - looks good (note: same changes to this file are also on Pioneer #13 pull request).

File: deployment/GP05MOAS-GL276_Deploy.csv [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+MV-1404,,TN-323,,GP05MOAS-GL276-02-DOSTAM000,1,1,2014-06-15T05:30:00,2015-06-02T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00012,50.07833,-144.80533,,1000,1000,
+MV-1404,,TN-323,,GP05MOAS-GL276-04-CTDGVM000,1,1,2014-06-15T05:30:00,2015-06-02T00:00:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-00104,50.07833,-144.80533,,1000,1000,
+SR1811,,TY-1902,,GP05MOAS-GL276-00-ENG000000,2,1,2018-07-23T06:35:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGCON-GLDROO-00276,50.1305,-144.5402,,1000,1000,
+SR1811,,TY-1902,,GP05MOAS-GL276-01-FLORDM000,2,1,2018-07-23T06:35:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-FLORTM-03260,50.1305,-144.5402,,1000,1000,FLORT deployed in place of customary FLORD
+SR1811,,TY-1902,,GP05MOAS-GL276-02-DOSTAM000,2,1,2018-07-23T06:35:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-DOSTAM-00145,50.1305,-144.5402,,1000,1000,
+SR1811,,TY-1902,,GP05MOAS-GL276-04-CTDGVM000,2,1,2018-07-23T06:35:00,2019-02-12T21:15:00,CGVGP-05MOAS-00000,CGVEH-GLDROO-00276,CGINS-CTDGVM-00104,50.1305,-144.5402,,1000,1000,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-GL276_Deploy.csv]: Reviewed by RT -looks good.

File: deployment/GP05MOAS-GL469_Deploy.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
-SR1812,,,,GP05MOAS-GL469-04-CTDGVM000,2,1,2018-08-22T01:04:05,,CGVGP-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,50.097617,-144.94278,,1000,1000,
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes
+SR1812,,SKQ20190514,,GP05MOAS-GL469-00-ENG000000,2,1,2018-08-22T01:04:05,2019-05-21T07:35:24,CGVGP-05MOAS-00000,CGVEH-GLDROO-00469,CGCON-GLDROO-00469,50.097617,-144.94278,,1000,1000,
+SR1812,,SKQ20190514,,GP05MOAS-GL469-01-FLORDM000,2,1,2018-08-22T01:04:05,2019-05-21T07:35:24,CGVGP-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-FLORDM-03528,50.097617,-144.94278,,1000,1000,
+SR1812,,SKQ20190514,,GP05MOAS-GL469-02-DOSTAM000,2,1,2018-08-22T01:04:05,2019-05-21T07:35:24,CGVGP-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-DOSTAM-00168,50.097617,-144.94278,,1000,1000,
+SR1812,,SKQ20190514,,GP05MOAS-GL469-04-CTDGVM000,2,1,2018-08-22T01:04:05,2019-05-21T07:35:24,CGVGP-05MOAS-00000,CGVEH-GLDROO-00469,CGINS-CTDGVM-09172,50.097617,-144.94278,,1000,1000,
```
Comment (rtravis-whoi) [inline deployment/GP05MOAS-GL469_Deploy.csv]: Reviewed by RT - looks good.

File: cruise/CruiseInformation.csv (hunk 1 above)
Comment (snwhite) [reply cruise/CruiseInformation.csv]: This looks cleaner than the version posted on the Pioneer Pull Request.
Looks good.

File: deployment/GP05MOAS-GL276_Deploy.csv (hunk 2 above)
Comment (snwhite) [inline deployment/GP05MOAS-GL276_Deploy.csv]: Looks good.

File: deployment/GP05MOAS-GL469_Deploy.csv (hunk 3 above)
Comment (snwhite) [reply deployment/GP05MOAS-GL469_Deploy.csv]: Looks okay – See comment on Sikuliaq Cruise Number above.

**Pull request merged**